Examples/Io/NuclearInteractions/src/detail/NuclearInteractionParametrisation.cpp

```diff
@@ -0,0 +1,341 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020-2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
I do not think you need to also place the implementation file in a `detail` subdirectory here. Can be placed directly in the `src` directory.

Reply:
It is not necessary, indeed. But at least for symmetry reasons compared to `include` we did it in several places.

Reply:
Ok, let's leave it as it is.